internal/config/config_test.go

```diff
@@ -18,15 +18,15 @@ func TestLoadConfig(t *testing.T) {
 	t.Run("succeeds", func(t *testing.T) {
 		t.Parallel()
 
-		c, err := config.LoadConfig(fakeDataDir, workingData)
+		c, err := config.LoadConfigWithFileLocation(fakeDataDir, workingData)
 		require.NoError(t, err)
 		require.Equal(t, "dummy_cluster", c.ClusterName)
 
 		t.Run("with_env_precedence", func(t *testing.T) {
 			_ = os.Setenv("NRI_KUBERNETES_CLUSTERNAME", "different_value")
 			_ = os.Setenv("NRI_KUBERNETES_NODENAME", "fake-node")
 
-			c, err := config.LoadConfig(fakeDataDir, workingData)
+			c, err := config.LoadConfigWithFileLocation(fakeDataDir, workingData)
```

Comment:
I would add a test for loadConfig as well

Reply:
Done.